data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -1,6 +1,8 @@
 # Suffixes
 PERCENTILE_FIELD_SUFFIX = " (percentile)"
 ISLAND_AREAS_PERCENTILE_ADJUSTMENT_FIELD = " for island areas"
+ADJACENT_MEAN_SUFFIX = " (including adjacency index)"
```

Comment:
I kinda hate these column names and stuff, but changed this to be a bit more copasetic with the repo's style